docs/modules/data-structures/pages/working-with-external-data.adoc

```diff
@@ -250,6 +250,57 @@ include::ROOT:example$/dds/map/PersonEntryStore.java[tag=personms]
 
 TIP: link:https://github.com/hazelcast/hazelcast-code-samples/tree/master/distributed-map/mapstore/src/main/java[See more MapStore/MapLoader code samples].
 
+== Offloading MapStore and MapLoader Operations
```

Comment:
@Serdaro you can have up-to-date info for the design here: https://github.com/hazelcast/hazelcast/blob/master/docs/design/map/map-store-operation-offloading.md

Reply:
@Serdaro also worth to mention the name of the executor for the offloaded calls and how oen can configure it: https://github.com/hazelcast/hazelcast/blob/master/docs/design/map/map-store-operation-offloading.md#configuring-offloadable-executor